README.md

```diff
@@ -1,5 +1,12 @@
-# graphql-query-whitelisting
-A simple GraphQL query whitelisting middleware for express
+# graphql-query-whitelist
+A simple GraphQL query whitelist toolkit for express.
+
+It includes:
+
+* An express middleware that prevents queries not in the whitelist to be executed. It also allows to execute queries just passing the queryId instead of the full query
```

Comment:
Respecto del queryId, redefiniría la oración así:

>It also allows to execute queries just passing a previously stored `queryId` instead of the full query.